.editorconfig

```diff
@@ -0,0 +1,78 @@
+root = true
+
+[*]
+indent_style = tab:suggestion
+insert_final_newline = true
+trim_trailing_whitespace = true:suggestion
+
+[*.cs]
+csharp_prefer_braces = true:suggestion
+csharp_new_line_before_open_brace = none:suggestion
```

Comment:
Invalid, you have `false` below.

Reply:
this is correct. `csharp_new_line_before_open_brace` accepts `all`, `none` or a list of accessors

Reply:
Then correct line below this 😉

Reply:
Done. didn't see there was a duplicate 😅